src/mlpack/tests/nystroem_method_test.cpp

```diff
@@ -146,7 +146,8 @@ TEST_CASE("GermanTest", "[NystroemMethodTest]")
 {
   // Load the dataset.
   arma::mat dataset;
-  data::Load("german.csv", dataset, true);
+  if (!data::Load("german.csv", dataset, true)) 
```

Comment:
```suggestion
  if (!data::Load("german.csv", dataset)) 
```
Third argument not required.